boxoffice/models/line_item.py

```diff
@@ -151,6 +177,10 @@ def fetch_all_details(cls, item_collection):
         line_item_query = db.select([cls.id, Order.invoice_no, Item.title, cls.base_amount, cls.discounted_amount, cls.final_amount, DiscountPolicy.title, DiscountCoupon.code, Order.buyer_fullname, Order.buyer_email, Order.buyer_phone, Assignee.fullname, Assignee.email, Assignee.phone, Assignee.details, OrderSession.utm_campaign, OrderSession.utm_source, OrderSession.utm_medium, OrderSession.utm_term, OrderSession.utm_content, OrderSession.utm_id, OrderSession.gclid, OrderSession.referrer]).select_from(line_item_join).where(cls.status == LINE_ITEM_STATUS.CONFIRMED).where(Order.item_collection == item_collection).order_by('created_at')
         return db.session.execute(line_item_query).fetchall()
 
+    @classmethod
+    def get_max_seq(cls, order):
+        return db.session.query(func.max(LineItem.line_item_seq)).filter(LineItem.order == order).first()[0]
```

Comment:
Yup, there is. Use that: http://docs.sqlalchemy.org/en/latest/orm/query.html?highlight=scalar#sqlalchemy.orm.query.Query.scalar